pom.xml

```diff
@@ -47,10 +47,13 @@
 
   <properties>
     <!-- alphabetized properties for versions -->
+    <version.guava>15.0</version.guava>
```

Comment:
Mixed thoughts, but i'm mostly in favor of keeping it.

One reason not to keep it: we're only using it for `@VisibleForTesting`.
But, in reality it's unlikely anyone outside Rocana uses this repo, and this is the Guava version Rocana uses, so why make things harder just in case?